File: src/app/models/person.ts
export class Person {
  personId: string;
  firstName: string;
  lastName: string;
  occupation: string;
  bio: string;
  avatarUrl: string;
  pictureUrl: string;

  constructor(personId: string, firstName: string, lastName: string, occupation: string, bio: string, avatarUrl: string, pictureUrl: string) {
    this.personId = personId;
    this.firstName = firstName;
    this.lastName = lastName;
    this.occupation = occupation;
    this.bio = bio;
    this.avatarUrl = avatarUrl;
    this.pictureUrl = pictureUrl;
  }
}

File: Models/Person.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace DataDisplayAPI.Models
{
    public class Person
    {
        [Key]
        public Guid PersonId { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Occupation { get; set; }
        public string Bio { get; set; }
        public string AvatarUrl { get; set; }
        public string PictureUrl { get; set; }
    }
}
